Nodejs/Product/Nodejs/Nodejs.csproj

```diff
@@ -585,6 +586,18 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
       <IncludeInVSIX>true</IncludeInVSIX>
     </Content>
+    <TypeScriptProject Include="ProjectTemplates\TypeScriptVuejsApp\TypeScriptVuejsApp.vstemplate">
```

Comment:
I don't think any of the item types here will result in the right zip files

Reply:
Fixed.